src/common/store/ui/share-modal/types.ts

```diff
@@ -1,39 +1,61 @@
 import { PayloadAction } from '@reduxjs/toolkit'
 
 import { ShareSource } from 'common/models/Analytics'
+import { Collection } from 'common/models/Collection'
 import { ID } from 'common/models/Identifiers'
 import { Track } from 'common/models/Track'
 import { User } from 'common/models/User'
 import { Nullable } from 'common/utils/typeUtils'
 
-export type ShareType = 'track' | 'profile'
+export type ShareType = 'track' | 'profile' | 'album' | 'playlist'
```

Comment:
Can you convert this to an enum instead and use ShareType.TRACK or ShareType.PROFILE, etc instead of the raw string values? :)